mappings/net/minecraft/entity/SpawnRestriction.mapping

```diff
@@ -1,13 +1,31 @@
 CLASS net/minecraft/class_1317 net/minecraft/entity/SpawnRestriction
 	CLASS class_1318 Entry
+		FIELD field_19349 predicate Lnet/minecraft/class_1317$class_4306;
 		FIELD field_6314 heightMapType Lnet/minecraft/class_2902$class_2903;
 		FIELD field_6315 location Lnet/minecraft/class_1317$class_1319;
 		METHOD <init> (Lnet/minecraft/class_2902$class_2903;Lnet/minecraft/class_1317$class_1319;Lnet/minecraft/class_1317$class_4306;)V
 			ARG 1 heightMapType
 			ARG 2 location
 	CLASS class_1319 Location
-	FIELD field_6313 mapping Ljava/util/Map;
-	METHOD method_20637 setRestrictions (Lnet/minecraft/class_1299;Lnet/minecraft/class_1317$class_1319;Lnet/minecraft/class_2902$class_2903;Lnet/minecraft/class_1317$class_4306;)V
+		METHOD <init> (Ljava/lang/String;I)V
+			ARG 1 name
+			ARG 2 id
+	CLASS class_4306 Predicate
```

Comment:
Not sure about naming this class Predicate as well. I guess it might be fine as its an innerclass?

Reply:
I named it `Predicate` because the unobfuscated `test` method is similar to `java.util.function.Predicate::test`. I don't think it'll conflict with the JDK one as that interface isn't used outside of SpawnRestriction.

Reply:
I renamed it to `SpawnPredicate`.